### PR TITLE
Part of lodash replacement. Replace omit

### PR DESCRIPTION
**Pull request by perunt**
Fixes RNBW-3658

## What changed (plus any additional context for devs)
Used custom omit instead of lodash one.
it's several times faster than lodash one

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels?
- [x] Added e2e tests? if not please specify why
- [x] If you added new files, did you update the CODEOWNERS file?

### CODE REVIEW

**Review by skylarbarrera — APPROVED** [review]

File: src/helpers/utilities.ts [hunk 1]
```diff
@@ -408,3 +408,19 @@ export const fromWei = (number: BigNumberish): string =>
 export const delay = (ms: number): Promise<void> => {
   return new Promise(resolve => setTimeout(resolve, ms));
 };
+
+/**
+ * @desc Can omit only flatted key, will not work with 'key.someObj.value'
```
Comment (tchayen) [inline src/helpers/utilities.ts:413]:
```suggestion
 * @desc Can omit only flattened key, will not work with 'key.someObj.value'
```

File: src/helpers/utilities.ts [hunk 2]
```diff
@@ -408,3 +408,19 @@ export const fromWei = (number: BigNumberish): string =>
...
+ */
+export const omitFlatten = <T extends object, K extends keyof T>(
+  obj: T,
+  keysToOmit: K[] | K
+): Omit<T, K> => {
+  const keysToUse = Array.isArray(keysToOmit) ? keysToOmit : [keysToOmit];
+  const _ = { ...obj };
+  keysToUse.forEach(key => {
+    // eslint-disable-next-line @typescript-eslint/no-dynamic-delete
+    delete _[key];
```
[the remark this answers is not in the document]
Comment (perunt) [reply src/helpers/utilities.ts]: You right. I got rid of the delete method. I made a few additional measurements with for..of and for..in, and the last one was the faster way.

File: src/helpers/utilities.ts [hunk 3]
```diff
@@ -408,3 +408,19 @@ export const fromWei = (number: BigNumberish): string =>
...
+
+/**
+ * @desc Can omit only flattened key, will not work with 'key.someObj.value'
+ */
+export const omitFlatten = <T extends object, K extends keyof T>(
+  obj: T | null | undefined,
+  keysToOmit: K[] | K
+): Omit<T, K> => {
+  const keys = Array.isArray(keysToOmit) ? keysToOmit : [keysToOmit];
+  const keysToUse = keys.map(String);
```
Comment (osdnk) [inline src/helpers/utilities.ts:420]: what's this?

**Review by osdnk — CHANGES_REQUESTED** [review]

File: src/helpers/utilities.ts (hunk 3 above)
Comment (perunt) [reply src/helpers/utilities.ts:420]: I tried to copy Lasha's behavior, it compared keys as it a string

File: src/components/context-menu/ContextMenu.android.js [hunk 4]
```diff
@@ -60,7 +60,9 @@ export default function ContextMenu({
           onPress={handleShowActionSheet}
           radiusAndroid={20}
         >
-          {children || <ContextButton {...omit(props, ActionSheetProps)} />}
+          {children || (
+            <ContextButton {...omitFlatten(props, ActionSheetProps)} />
+          )}
```
Comment (tchayen) [inline src/components/context-menu/ContextMenu.android.js:65]: Oo, so here you change that part from another PR that I commented on 😅.

**Review by tchayen — APPROVED** [review]
Looks good to me. Great that you did benchmarks for those different variants.

File: src/components/context-menu/ContextMenu.android.js (hunk 4 above)
Comment (perunt) [reply src/components/context-menu/ContextMenu.android.js:65]: usually, I don't change the old naming, but yea, in this case, it makes sense to update it

**Review by osdnk — APPROVED** [review]

**Review by terrysahaidak — APPROVED** [review]

**Review by ibrahimtaveras00 — APPROVED** [review]
👍🏽

**Pull request merged**